app/models/deletion.rb

```diff
@@ -30,7 +30,7 @@ def record_metadata
   end
 
   def expire_api_memcached
-    ["deps/v1/#{rubygem}", "info/#{rubygem}", "versions", "names"].each do |key|
+    ["deps/v1/#{rubygem}", "versions", "names"].each do |key|
```

Comment:
right, when the page gets requested before the transaction that sets the yank is committed, the gem won't show as yanked yet, so fastly will then re-cache the page with the unyanked version